chapters/classes.tex

```diff
@@ -202,7 +202,7 @@ \subsection{Component Declaration Static Semantics}\label{component-declaration-
 
 \subsubsection{Declaration Equations}\label{declaration-equations}
 
-An environment that defines the value of a component of built-in type is said to define a \firstuse{declaration equation}\index{declaration equation}\index{declaration equation!zzzzz@\igobble |seealso{binding equation}} associated with the declared component.
+An environment that defines the value of a component of built-in type is said to define a \firstuse{declaration equation}\index{declaration equation!zzzzz@\igobble |seealso{binding equation}} associated with the declared component.
```

Comment:
Should this mechanism for adding see also be documented in the styleguide as well? Or is it something that everybody knows?

Reply:
To me, this is more of a technicality of the _index.sty_ (or _makeidx.sty_) package than content for a style guide.  But maybe our style guide needs to be both a style guide and a technical guide for how to write LaTeX code?

I mean, describing how to write the source code in order to maintain good style is one thing that has a clear place in the style guide.  It's when it becomes technical about things not directly related to style that I hesitate.

Reply:
By the way, this is not a trick I assume everybody knows, and it took me some time to find it.  However, I'm afraid that the LaTeX world (and even just the packages more or less supported by LaTeXML) is an endless source of the need for such tricks.  If we had a document with our own selection of tips and tricks, would anyone read it?  My guess is that most authors would just look for similar situations in existing code of the document, and not even remember that there's a collection of tips and tricks.  A style guide, on the other hand, has a normative role and gives us a place for having discussions about what we want the document to look like.

Reply:
> By the way, this is not a trick I assume everybody knows, and it took me some time to find it. 

That's why I asked. It had the look of a trick/hack that required some labor to come up with.

> However, I'm afraid that the LaTeX world (and even just the packages more or less supported by LaTeXML) is an endless source of the need for such tricks. If we had a document with our own selection of tips and tricks, would anyone read it? My guess is that most authors would just look for similar situations in existing code of the document, and not even remember that there's a collection of tips and tricks. A style guide, on the other hand, has a normative role and gives us a place for having discussions about what we want the document to look like.

That is a good point. There are not so many people who currently author the content, unfortunately. ( Ahem, guilty myself.) Maybe we need to first have a broader discussion of how to attract more people to making the changes and what is holding them up? @HansOlsson, should we put it on the agenda for the next Phone meeting?

Reply:
Would make sense to discuss yes, but I would think that other issues that LaTeX-hacks are more blocking.
When adding them I generally add references to them (e.g. to tex.stackexchange.com).

I also think that we should avoid such hacks when possible  - and an alternative in this case would be that the text of "Declaration Equation" section would say that it is a sub-set of "Binding Equations" with reference; instead of using the hack to put "see also" in the index.

Reply:
Can we resolve this and continue in https://github.com/modelica/ModelicaSpecification/pull/3043#discussion_r757120747 to try to keep the discussion in one place?

Reply:
Resolving to continue the discussion in the comment as suggested above.